modules/core/src/main/scala/org/tessellation/domain/trust/storage/OrdinalTrustMap.scala

```diff
@@ -0,0 +1,35 @@
+package org.tessellation.domain.trust.storage
+
+import cats.Order
+import cats.syntax.contravariant._
+
+import org.tessellation.cli.method.snapshotInterval
+import org.tessellation.schema.SnapshotOrdinal
+
+import derevo.circe.magnolia.encoder
+import derevo.derive
+
+@derive(encoder)
+case class OrdinalTrustMap(
+  ordinal: SnapshotOrdinal,
+  peerLabels: PublicTrustMap,
+  trust: TrustMap
```

Comment:
`trust` is of type `TrustMap` to make it easy to update the `next` `OrdinalTrustMap` every `snapshotInterval` ordinals. We will never use `trust.peerLabels` for our trust calculation as of certain ordinals. We will only use `peerLabels`.

Reply:
and what's a difference between those two?

Reply:
Ok @cngo-github but what's the difference between `peerLabels` and `trust.peerLabels`?

Reply:
`trust.peerLabels` are the ones copied over from `trust` when `trust` was copied. It is not used by `OrdinalTrustMap`. `peerLabels` have the ordinal trust data received via gossip from other nodes. It matters to `OrdinalTrustMap`.

Reply:
ok, so why you need to store `trust.peerLabels` if they're not used by `OrdinalTrustMap`?

Reply:
`trust` is a copy of the real-time deterministic trust storage at a given ordinal. To preserve the integrity of that copy, it was decided to copy everything over, just in case.